package.json

```diff
@@ -36,7 +36,7 @@
     }
   ],
   "dependencies": {
-    "react-native-tabs": "^1.0.2"
+    "react-native-tab-navigator": "joenoon/react-native-tab-navigator#master"
```

Comment:
its not currently possible to do certain styling with the npm version of this lib.  i am waiting on a PR https://github.com/exponentjs/react-native-tab-navigator/pull/73 but it doesnt look like its very high priority over there.  are there big concerns with leaving a github dep in?